cmd/notifyPendingCI.go

```diff
@@ -27,7 +27,7 @@ var notifyPendingCICmd = &cobra.Command{
 		var listPendingPRs []string
 		for {
 			log.Println("Checking for community PRs that are waiting for CI to run")
-			listPendingPRs, err = github2.CheckForPendingCI(client, cfg.InternalTeam, cfg.CheckStalePending)
+			listPendingPRs, err = github2.CheckForPendingCI(client, cfg.InternalTeam, cfg)
```

Comment:
This comment should be taken universally for all 3 of these different cmd modes,  but since you pass cfg, you no longer also need to pass cfg.InternalTeam, but you will also need to change references to this parameter to cfg.InternalTeam in the functions that previously used this string var directly from the function parameters.

Reply:
fixed